### PR TITLE
Update electronic-prescription-service-api.yaml

### PR DESCRIPTION
**Pull request by tonyheap**

### CODE REVIEW

File: specification/electronic-prescription-service-api.yaml [hunk 1]
```diff
@@ -307,6 +311,7 @@ paths:
             application/fhir+json:
               schema:
                 type: object
+                description: Confirmation that the prescriptioon has been created, expressed as a [FHIR OperationOutcome resource](https://www.hl7.org/fhir/operationoutcome.html).
```
Comment (ChrisLopezUK) [inline specification/electronic-prescription-service-api.yaml:314]: Typo in prescription

**Review by ChrisLopezUK — CHANGES_REQUESTED** [review]
Thanks Tony.

Only found a minor typo other than that it looks good.

File: specification/electronic-prescription-service-api.yaml [hunk 2]
```diff
@@ -489,6 +498,10 @@ paths:
           application/fhir+json:
             schema:
               type: object
+              description: |
+                The prescription orders to be verified, including the signatures,
+                expressed as a FHIR bundle of [prescription-order-update FHIR messages](https://simplifier.net/guide/NHSDigital-Medicines/Home/FHIRAssets/AllAssets/MessageDefinition-duplicate-2/prescription-order-update.guide.md).
```
Comment (dan-jones-iw) [inline specification/electronic-prescription-service-api.yaml:503]: think [this](https://simplifier.net/guide/NHSDigital-Medicines/Home/FHIRAssets/AllAssets/Profiles/NHSDigital-Task.guide.md?version=current) link matches the other ones and is appropriate for `return` / `withdraw` interactions

File: specification/electronic-prescription-service-api.yaml [hunk 3]
```diff
@@ -550,6 +563,9 @@ paths:
         required: true
         content:
           application/fhir+json:
+            schema:
+              type: object
+              description: The request to return a downloaded prescription, expressed as a [Task FHIR resource](https://simplifier.net/resolve?target=simplifier&scope=uk.nhsdigital.r4&canonical=https://fhir.nhs.uk/StructureDefinition/NHSDigital-Task).
```
Comment (dan-jones-iw) [inline specification/electronic-prescription-service-api.yaml:568]: same as above

File: specification/electronic-prescription-service-api.yaml [hunk 4]
```diff
@@ -681,6 +700,9 @@ paths:
         required: true
         content:
           application/fhir+json:
+            schema:
+              type: object
+              description: The request to withdraw a dispense notification, expressed as a [Task FHIR resource](https://simplifier.net/resolve?target=simplifier&scope=uk.nhsdigital.r4&canonical=https://fhir.nhs.uk/StructureDefinition/NHSDigital-Task).
```
Comment (dan-jones-iw) [inline specification/electronic-prescription-service-api.yaml:705]: same as above

File: specification/electronic-prescription-service-api.yaml [hunk 5]
```diff
@@ -812,6 +837,7 @@ paths:
             application/fhir+json:
               schema:
                 type: object
+                description: TBC
```
Comment (dan-jones-iw) [inline specification/electronic-prescription-service-api.yaml:840]: do we need to address the TBC now?

File: specification/electronic-prescription-service-api.yaml (hunk 1 above)
Comment (tonyheap) [reply specification/electronic-prescription-service-api.yaml]: Fixed

File: specification/electronic-prescription-service-api.yaml (hunk 3 above)
Comment (tonyheap) [reply specification/electronic-prescription-service-api.yaml]: Fixed

File: specification/electronic-prescription-service-api.yaml (hunk 4 above)
Comment (tonyheap) [reply specification/electronic-prescription-service-api.yaml]: Fixed. Also changed similar link in the endpoint description.

File: specification/electronic-prescription-service-api.yaml (hunk 2 above)
Comment (tonyheap) [reply specification/electronic-prescription-service-api.yaml:503]: This is for the verify signature function, not for return or withdraw. So I don't think your comment is applicable.

File: specification/electronic-prescription-service-api.yaml (hunk 5 above)
Comment (tonyheap) [reply specification/electronic-prescription-service-api.yaml]: Good spot - I've removed that line for now.

File: specification/electronic-prescription-service-api.yaml (hunk 2 above)
Comment (dan-jones-iw) [reply specification/electronic-prescription-service-api.yaml:503]: oops comment in wrong place apologies

Comment (dan-jones-iw) [reply specification/electronic-prescription-service-api.yaml:503]: however, i think its inaccurate to say that the Bundle is a Bundle of `prescription-order-update` messages. the release response message that goes to this endpoint is a Bundle of prescriptions (`prescription-order` messages). think [this](https://simplifier.net/guide/NHSDigital-Medicines/Home/FHIRAssets/AllAssets/MessageDefinition-duplicate-2/prescription-order.guide.md) is the correct link

Comment (tonyheap) [reply specification/electronic-prescription-service-api.yaml]: Well spotted, sorry about that. Changed now, and for completeness I have added a similar description to the 200 response from the download prescriptions endpoint - which is essentially the same payload, I believe.

**Review by ChrisLopezUK — APPROVED** [review]

**Pull request merged**